ios/browser/api/certificate/enums/brave_certificate_enums.h

```diff
@@ -0,0 +1,177 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_CERTIFICATE_ENUMS_BRAVE_CERTIFICATE_ENUMS_H_
+#define BRAVE_IOS_BROWSER_API_CERTIFICATE_ENUMS_BRAVE_CERTIFICATE_ENUMS_H_
+
+#import <Foundation/Foundation.h>
+
+#if defined(__cplusplus)
+#import "brave/ios/browser/api/common/brave_foundation.h"
+#else
```

Comment:
Is is not possible for us to import with `__cplusplus` defined in iOS through some local Obj-C++? 

We should probably just use `#include "brave_foundation.h"` regardless